tests/Maester/Entra/Test-MtEntraIDConnect.Tests.ps1

```diff
@@ -1,49 +1,9 @@
 ﻿BeforeDiscovery {
-
+    $EntraIDPlan = Get-MtLicenseInformation -Product "EntraID"
```

Comment:
The only issue here is that test discovery fails if we are not yet connected to Microsoft Graph. (Try running `Get-MtTestInventory` before `Connect-Maester.`) Can we wrap the license check in a try/catch block or silently continue on error?

Reply:
Fixed this in this and two other test files. Now the command is run without issues